index.css

```diff
@@ -115,7 +115,7 @@ input {
     border-bottom:1px solid #000;
 
 }
-
+/* Love the color scale */
```

Comment:
I was actually thinking about making a scale algorithm and use inline styles so instead of just 6 colors it would be a spectrum. However that would be an overkill for this so I took the easier route that has basically the same UX.